csharp/extractor/Semmle.Extraction.CSharp.DependencyFetching/DependencyContainer.cs

```diff
@@ -68,4 +68,18 @@ public void AddFramework(string framework)
             Packages.Add(GetPackageName(p));
         }
     }
-}
\ No newline at end of file
+
+    internal static class DependencyContainerExtensions
+    {
+        /// <summary>
+        /// Flatten a list of containers into a single container.
+        /// </summary>
+        public static DependencyContainer Flatten(this IEnumerable<DependencyContainer> container) =>
```

Comment:
You could change this to
```csharp
 public static DependencyContainer Flatten(this IEnumerable<DependencyContainer> container, DependencyContainer init) =>
```
and then replace the `new DependencyContainer()` in `Aggregate` below with `init`.